libraries/stdlib/stdlib_ng.mtlx

```diff
@@ -43,7 +43,7 @@
       <parameter name="frameoffset" type="integer" interfacename="frameoffset"/>
       <parameter name="frameendaction" type="string" interfacename="frameendaction"/>
     </image>
-    <output name="N_out_float" type="float" nodename="N_img_float"/>
+    <output name="out" type="float" nodename="N_img_float"/>
```

Comment:
The output names did not match the corresponding nodedef in stdlib_defs, where the outputs are declared explicitly now.